draft-ietf-tls-tls13.md

```diff
@@ -2306,9 +2387,11 @@ values, each representing a single set of key agreement parameters;
 for instance a client might offer shares for several elliptic curves
 or multiple integer DH groups. The shares for each ClientKeyShareOffer
 MUST by generated independently. Clients MUST NOT offer multiple
-ClientKeyShareOffers for the same parameters. It is explicitly
-permitted to send an empty client_key_share extension as this is used
-to elicit the server's parameters if the client has no useful
+ClientKeyShareOffers for the same parameters. Servers receiving duplicate
+ClientKeyShareOffers SHOULD respond with an "illegal_parameter" alert
```

Comment:
An exact duplicate doesn't hurt; it's just waste. There's really no point in forcing implementations to check past the point where they find what they need. I'm not strongly against a MUST here, though.